docs/tutorials/launch-jupyter-lab-tutorial.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 2
+---
+
+# Jupyter Lab Notebook Tutorial
+
+
+In this tutorial we will show how to open Jupyter Lab notebook
+on your computer using executor rented from fair. This way
+you can leverage your GPU to run inference and avoid paying for expensive
+cloud compute.
+
+## Pulling the Jupyter Docker Image
```

Comment:
This section is not needed as `docker run` command will automatically pull the image. Additionally, `docker pull` command works locally, so if you're running this on a remote machine - it will not work.